auth_session_timeout/i18n/de.po

```diff
@@ -1,23 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
You need to remove POT files, not PO ones. Having PO files, although outdated, allow to reuse the sentences that haven't changed. Next Transifex synchronization will wipe out the non existing terms comparing with the newest POT file.

@atchuthan proposed in https://github.com/OCA/maintainer-tools/issues/232 that we should avoid the commits that updates the languages in the migration process, but I'm evaluating if this is a good idea at all.